### DISCUSSION TITLE
Addressing second charger with set_charge_rate service.

### DISCUSSION THREAD

**Discussion by rbb-dev** [lbbrhzn/ocpp#1399] [General]
It seems that set_charge_rate can only address the last charger configured/connected.
Can the set_charge_rate service be changed also to allow specifying the charger serial and, if not sent, using the current behaviour?
This would allow us to address the second charger and won't break the current installs.
Something like this for example:
limit_watts: 10000
conn_id: 1
charger: 3e95

Or is there a better way?
or maybe registering the services under domain other than ocpp?
Maybe register them under same domain as the start charging buttons?

**Comment by rbb-dev** [comment]
Bump?

**Comment by rbb-dev** [comment]
Anyone?
When you use more than 1 charger, how can you use set_charge_rate for each charger??
Please?

**Comment by lightheaded** [comment]
Multiple chargers are not supported as of now (#803). I also have two chargers at home, and would like to get them both connected. I am waiting for #1381 to be completed until contributing to #803.

**Reply by rbb-dev** [reply]
@lightheaded I'd be happy to help you test it when you start on it.

**Reply by LeoAlioth** [reply]
I am joining in with both development and testing if needed

**Reply by lightheaded** [reply]
Hey! I'll have to take a backseat on this, as it's once again a busier time period in my life. However, whoever ends up leading the project, please feel free to tag me so I can potentially help.

Also, I got my use-case working by just adding two integrations on different ports. This solves my immediate problem, albeit not very elegantly.

**Comment by drc38** [comment]
See https://github.com/lbbrhzn/ocpp/pull/1498 for testing with multiple chargers
